cloudferry_devlab/cloudferry_devlab/tests/config.py

```diff
@@ -61,6 +61,10 @@
 """This file contains map of relationships between external networks on source
 and destination clouds."""
 
+case_sensitivity_test_user = 'user9'
```

Comment:
Its a variable referenced in dst_users. Currently not supported as a list in test design. Added description.